packages/node_modules/@webex/internal-plugin-devices/test/integration/spec/devices.js

```diff
@@ -374,20 +364,17 @@ describe('plugin-devices', () => {
         });
 
         describe('when the device is not registered', () => {
-          beforeEach('check that the device is not registered', () => {
-            assert.isFalse(devices.registered);
-          });
-
           describe('when the device successfully registers', () => {
-            it('should resolve the promise with the websocket url', () => Promise.all([
-              devices.getWebSocketUrl(wait),
-              devices.register()
-            ])
-              .then(([url]) => {
-                assert.isDefined(url);
-                assert.isTrue(services.isServiceUrl(url));
-                assert.include(url, 'mercury');
-              }));
+            it('should resolve the promise with the websocket url',
+              () => Promise.all([
+                devices.getWebSocketUrl(wait),
+                devices.register()
+              ])
+                .then(([url]) => {
+                  assert.isDefined(url);
```

Comment:
This is covered by the other assertions.

Reply:
Removed this assertion.